cmd/cape/cmd/deploy.go

```diff
@@ -184,7 +169,20 @@ func doDeploy(url string, functionInput string, functionName string, insecure bo
 		return "", nil, err
 	}
 
-	id, hash, err := Deploy(url, token, functionName, reader, insecure, pcrSlice)
+	functionTokenPublicKey, err := getFunctionTokenPublicKey()
```

Comment:
I like how you kept this in the CLI package :)

Reply:
anything interacting with the file system didn't strike me as SDK material :)